grading/ssh-sudoer/bin/INGInious

```diff
@@ -0,0 +1,435 @@
+#!/bin/python3
```

Comment:
It is unthinkable that we merge a bunch of copy/pasted files with some minors changes in it. This would make maintenance really difficult and it is already not simple right now. As @GuillaumeDerval suggests not to include this container specificities into the base container, I recommend working with hooks or plugins. We can think of a folder containing the hooks that the main ``INGInious`` script will open and execute at specific times.

I won't even provide more feedback on this until this is clean.